tests/integration/concurrent_transaction_test/test_concurrent_transaction.py

```diff
@@ -1,136 +0,0 @@
-from tests.integration.base import DBTIntegrationTest, use_profile
-import threading
-from dbt.adapters.factory import FACTORY
-
-
-def get_adapter_standalone(config):
-    plugin = FACTORY.plugins[config.credentials.type]
-    cls = plugin.adapter
-    return cls(config)
```

Comment:
does the new functional test need this?

Reply:
I tried adding it but it didn't make a difference. The error was in the underlying udf so I wonder if it might be a race condition.